tests/unit/components/sl-date-picker-test.js

```diff
@@ -0,0 +1,264 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-date-picker', 'Unit | Component | sl date picker', {
+    unit: true
+});
+
+test('it renders', function( assert ) {
+    assert.expect( 2 );
+
+    // Creates the component instance
+    var component = this.subject();
+    assert.equal( component._state, 'preRender' );
+
+    // Renders the component to the page
+    this.render();
+    assert.equal( component._state, 'inDOM' );
+});
+
+test( 'Default class names are present', function( assert ) {
```

Comment:
Either in this test or different one the classes applied to the ``{{input}}`` in the template need to be ensured as well.